sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -399,6 +397,9 @@ def upload_blob(  # pylint: disable=too-many-locals
         :param metadata:
             Name-value pairs associated with the blob as metadata.
         :type metadata: dict(str, str)
+        :keyword blob_tags:
+            Name-value pairs associated with the blob as tag.
+        :paramtype blob_tags: dict(str, str)
```

Comment:
curious - is :paramtype rigjt? though it should just be :type

Reply:
There was a bug previously and :paramtype was recommended to use when document a keyword type, I'm not sure if the bug is fixed now...

Reply:
Yes I think paramtype is correct.